java/lib/src/main/java/com/svix/Webhook.java

```diff
@@ -35,6 +35,14 @@ public Webhook(final String secret) {
 		}
 		this.key = Base64.getDecoder().decode(sec);
 	}
+	
+	public Webhook(final byte[] secret) {
+		this.key = secret;
+	}
+	
+	public WebhookFromRawString(final String secret) {
+		this.key = secret.getBytes();
+	}
```

Comment:
Webhook should remain as is, and then you should have `WebhookRaw` that has two variants, the raw string and raw bytes.